programs/openbook_v2/tests/integration.rs

```diff
@@ -0,0 +1,319 @@
+     let (bids, asks, event_heap) = create_bids_asks_event_heap(&mut banks_client, &keypair).await;
+    let (market,
+        event_authority,
+        market_authority,
+        market_base_vault,
+        market_quote_vault,
+    ) = create_default_market(&mut banks_client, &keypair, &base_mint, &quote_mint, &bids, &asks, &event_heap).await?;
+    let market_keys = MarketKeys{
+        market,
+        bids,
+        asks,
+        event_heap,
+        event_authority,
+        market_authority,
+        market_base_vault,
+        market_quote_vault,
+    };
+    // create open_orders_account ( and open_orders_indexer too)
+    let (ooa, _ooi) =  setup_open_orders_account(&mut banks_client, &keypair, &market ).await?;
+    // market maker place bid
+    place_order_and_execute(&mut banks_client,
+                            &keypair,
+                            PlaceOrder        {
+                                side: Side::Bid,
+                            price_lots: 150_000,
+                            max_base_lots: 1000,
+                            max_quote_lots_including_fees: 500_000_000,
+                            client_order_id: 0,
+                            order_type: PlaceOrderType::Limit,
+                            expiry_timestamp: 0,
+                            self_trade_behavior: SelfTradeBehavior::DecrementTake,
+                            limit: 10,
+                            },
+                            &market_keys,
+                            &ooa,
+                            &quote_mint,
+    ).await?;
+    // market maker place ask
+    place_order_and_execute(&mut banks_client,
+                            &keypair,
+                            PlaceOrder{
+                                side: Side::Ask,
+                                price_lots: 154_000,
+                                max_base_lots: 1000,
+                                max_quote_lots_including_fees: 500_000_000,
+                                client_order_id: 0,
+                                order_type: PlaceOrderType::Limit,
+                                expiry_timestamp: 0,
+                                self_trade_behavior: SelfTradeBehavior::DecrementTake,
+                                limit: 10,
+                            },
+                            &market_keys,
+                            &ooa,
+                            &base_mint,
+    ).await?;
+    // init drift
+    let (state, drift_signer) = initialize_drift(&mut banks_client, &keypair, &quote_mint).await?;
+    println!("state: {} drift signer: {}", state, drift_signer);
+    // init quote market
+    let (quote_market, quote_market_vault, quote_insurance_fund_vault ) = init_quote_market(
+        &mut banks_client,
+        &keypair,
+        &quote_mint,
+        &drift_signer,
+        &state,
+    ).await?;
+
+    // pyth feed
+    let oracle_feed = initialize_pyth_oracle(&mut banks_client, &keypair,  PythInitialize{
```

Comment:
would prob be easy to make this static and use `ProgramTest::load_account_with_base64_data` instead

Reply:
yes, you are correct, but saw it in your anchor tests and imported it from there